public/app/themes/clarity/inc/admin/plugins/wp-elasticsearch.php

```diff
@@ -65,33 +65,65 @@ public function removeFilterWeightRecent($formatted_args, $args)
             return $formatted_args;
         }
 
+        // The following code is a copy/paste from an ElasticPress query.
+        // "functions": [
+        //     {
+        //         "exp": {
+        //             "post_date_gmt": {
+        //                 "scale": "14d",
+        //                 "decay": 0.25,
+        //                 "offset": "7d"
+        //             }
+        //         }
+        //     },
+        //     {
+        //         "weight": 0.001
+        //     }
+        // ],
```

Comment:
Should we remove this?

Reply:
With OpenSearch json being a new language to me, I'd prefer to keep it. 

I feel like it could be useful in the future to understand the developer's thought process.

Reply:
I'll send it through, and delete at a later date if needs be.